settings.py

```diff
@@ -61,6 +61,8 @@
 
 INFLUXDB_STRATAGEM_SCAN_DB = "iml_stratagem_scans"
 
+INFLUXDB_PORT = 8086
```

Comment:
This is odd because the port here does not actually configure anything, it's just reflecting the default value of influxdb.

In other words changing this would not actually change the config of influx.

Reply:
I was using it in three places in https://github.com/whamcloud/integrated-manager-for-lustre/blob/GH-852/chroma_core/lib/stratagem.py#L93

Reply:
I can hardcode it if you prefer

Reply:
Not commenting on DRYness but the fact that this setting is not configuring influxdb which is different than all the other settings in this file.

Reply:
will hardcode